R/cv_varsel.R

```diff
@@ -203,7 +204,8 @@ cv_varsel.vsel <- function(
               "brms:::get_refmodel.brmsfit() to some non-`NULL` value.")
     }
   }
-  return(cv_varsel(
+
+  return(do.call(cv_varsel, c(list(
```

Comment:
I avoided `do.call()` intentionally in case of large arguments (here, the `refmodel` is typically large), see commit 4c0a5b2e16120473665fa2feda1679af8e4d6c24.

Reply:
If I understood correctly, you are saying use of `do_call()` is fine here (previously there was no `do_call()` here

Reply:
`do_call()` should probably be fine here, but the statement from the `brms::do_call()` docs (https://github.com/paul-buerkner/brms/blob/ceffe5353730c769a1506f5b1a01110b5ba2c36d/R/misc.R#L659-L661) that `do_call()` might be inefficient in case of a large number of arguments led me to the decision to use direct function calls wherever possible (I have also added a corresponding item to my projpred to-do list: https://github.com/fweber144/todo_projpred/blob/00041501e545d9c295c7839fd310dc8329bb49ea/to-do.Rmd#L9).

Reply:
Direct function call seems to make it less flexible when calling with previous vsel object, which makes it annoying for users (I did get annoyed). Thus, I would use do_call if possible, or other way to pass the previous arguments